src/js/rtp-stats.js

```diff
@@ -7,74 +7,81 @@
+                        procMilliseconds:   is_defined(parseInt(statsReport.stat('googCurrentDelayMs'))),
+                        rttMilliseconds:    when_defined(parseInt(statsReport.stat('googRtt'))),
+                        jbMilliseconds:     when_defined(parseInt(statsReport.stat('googJitterReceived')))
                     };
 
-                } else if (is_defined(statsReport.packetsReceived) && statsReport.mediaType == 'audio' && streamType === 'audio_output') {
+                } else if (parseInt(statsReport.stat('packetsReceived')) && statsReport.stat('mediaType') == 'audio' && streamType === 'audio_input') {
                     extractedStats = {
                         timestamp:          timestamp,
-                        packetsCount:       statsReport.packetsReceived,
-                        bytesReceived:      statsReport.bytesReceived,
-                        audioLevel:         when_defined(statsReport.audioOutputLevel),
-                        packetsLost:        is_defined(statsReport.packetsLost) ? Math.max(0, statsReport.packetsLost) : 0,
-                        procMilliseconds:   is_defined(statsReport.googCurrentDelayMs),
-                        jbMilliseconds:     when_defined(statsReport.googJitterBufferMs)
+                        packetsCount:       parseInt(statsReport.stat('packetsReceived')),
+                        bytesReceived:      parseInt(statsReport.stat('bytesReceived')),
+                        audioLevel:         when_defined(parseInt(statsReport.stat('audioOutputLevel'))),
+                        packetsLost:        is_defined(parseInt(statsReport.stat('packetsLost'))) ? Math.max(0, statsReport.stat('packetsLost')) : 0,
+                        procMilliseconds:   is_defined(parseInt(statsReport.stat('googCurrentDelayMs'))),
+                        jbMilliseconds:     when_defined(parseInt(statsReport.stat('googJitterReceived')))
                     };
 
                 } else if (is_defined(statsReport.packetsSent) && statsReport.mediaType == 'video' && streamType === 'video_input') {
                     extractedStats = {
                         timestamp:          timestamp,
-                        packetsCount:       statsReport.packetsSent,
-                        bytesSent:          statsReport.bytesSent,
-                        packetsLost:        is_defined(statsReport.packetsLost) ? Math.max(0, statsReport.packetsLost) : 0,
-                        rttMilliseconds:    when_defined(statsReport.googRtt),
-                        procMilliseconds:   is_defined(statsReport.googCurrentDelayMs),
-                        frameRateSent:      when_defined(statsReport.googFrameRateSent),
+                        packetsCount:       parseInt(statsReport.stat('packetsSent')),
+                        bytesSent:          parseInt(statsReport.stat('bytesSent')),
+                        audioLevel:         when_defined(parseInt(statsReport.stat('audioInputLevel'))),
```

Comment:
audio level?

Reply:
This was supposed to be round trip time. Fixed now.